### DISCUSSION TITLE
how to build this project

### DISCUSSION THREAD

**Discussion by zzRaven** [noonchen/STDF-Viewer#115] [Q&A] [answered]
Hi Sir

Iam new to cython，I would be appreciate if you tell me how to complie the Cython and stdf-viewer.

Thanks a lot!

**Comment by noonchen** [comment]
Hi @zzRaven ,

there will be a new version in the next month, the building process will be much easier than Cython, and I'll add the process in the readme, please stay tuned.

thanks!

**Comment by Cyril-Marti** [comment]
Hi Noon Chen,

Do you have news about the process to build the tool?

I've compiled the rust-stdf library and the rust_stdf_helper with rust.
But I'm now stuck (python can't find the rst_stdf_helper anyway) 😥

Do you have some quick hints to help me?

Thank you

**Reply by noonchen** [reply]
Hi @zzRaven @Cyril-Marti ,
The steps are documented in README~

**Reply by Cyril-Marti** [reply]
Hi Noon Chen,

I have check on the documents but the only README I've found is the one at the root README.md, but I can't see any build instructions.
Do I miss them? Or is it on a branch?

Thx

**Reply by noonchen-ni** [reply, marked as answer]
@Cyril-Marti I think you missed it, here is the link: https://github.com/noonchen/STDF-Viewer#build

**Reply by Cyril-Marti** [reply]
Hi @noonchen,
Thank you very much. I indeed missed it - my database was not updated from the last changes.
Sorry for that.
